extension_config.cmake

```diff
@@ -1,5 +1,6 @@
 # This file is included by DuckDB's build system. It specifies which extension to load
 
+duckdb_extension_load(icu)
```

Comment:
Note: `icu` is added because we're adding a test for `timestamptz`, for timezones we need `icu` in duckdb